articles/hdinsight/hdinsight-apps-publish-applications.md

```diff
@@ -56,14 +56,10 @@ There are two steps involved for publishing applications to the Azure Marketplac
 | tiers |The cluster tiers that the application is compatible with. |Standard, Premium, (or both) |
 | versions |The HDInsight cluster types that the application is compatible with. |3.4 |
 
-## Package application
-Create a zip file that contains all required files for installing your HDInsight applications. You will need the zip file in [Publish application](#publish-application).
-
-* [createUiDefinition.json](#define-application).
-* mainTemplate.json. See a sample at [Install custom HDInsight applications](hdinsight-apps-install-custom-applications.md).
-  
+## Application install script
+The HDInsight application will be installed by executing an application install script on an edge node. 
```

Comment:
IMO this is still a little confusing. Even if we don't elaborate on what an 'edge node' is, we need to convey that an edgenode is provisioned for the application (and that is where the installer script runs)